### ISSUE TITLE
Content Item Cloneable property

### ISSUE THREAD

**Issue by urbanit** [OrchardCMS/Orchard#6384]
Is there any way to prevent a content item from being cloned? 
Maybe having a setting "Cloneable" alongside Creatable, Listable, ...

**Comment by CSurieux** [comment]
Yes ... and the default value should be no.

**Comment by timmahon** [comment]
Agreed that there should be a setting to prevent cloning of a content item, however as a workaround, you can hide the Clone link in the Content List for a particular Content item in Placement.info 

```
<Match ContentType="TypeThatYouDontWantToClone">
    <Match DisplayType="SummaryAdmin">
      <Place Parts_Contents_Clone_SummaryAdmin="-"/>     
    </Match>
</Match>
```

**Comment by urbanit** [comment]
Good to share

**Comment by bleroy** [comment]
Why should the default value be no?

**Comment by CSurieux** [comment]
It is not a catholic point of view but so many problem cloning elements which are badly prepared to cloning in one of their parts . And again the DB is broken,   often easy to repair on a dev db but on a large production DB....

**Comment by bleroy** [comment]
Please file bugs for these specific issues that happen for badly prepared elements.

**Comment by CSurieux** [comment]
They are in modules, remember there is an Orchard Gallery :)

**Comment by bleroy** [comment]
Right, I understand that. What causes them be badly cloned?
